qemu/tests/cgroup.py

```diff
@@ -1703,7 +1705,7 @@ def _get_stat(pid):
 
         timeout = int(params.get("login_timeout", 360))
         vm = env.get_all_vms()[0]
-        vm_cpus = int(params.get('smp', 0))     # cpus per VM
+        vm_cpus = vm.cpuinfo.smp  # cpus per VM
```

Comment:
Suggest a separate commit

Reply:
@yanan-fu I need to handle this conflict. My branch master is not the latest.  This patch is merged before.